Modelica/Mechanics/Translational/Components/ElastoGap.mo

```diff
@@ -116,6 +119,16 @@ Note, since |f_d| &le; |f_c|, pulling forces cannot occur and the contact force
 is always continuous, especially around the start of the penetration at s_rel = s_rel0.
 </p>
 
+<p>
+In order to have consistent units for non-linear springs the formula <code>c*|s_rel|^n</code>
+is replaced by <code>f_ref*|s_rel/s_ref|^n</code>, where as default <code>s_ref&nbsp;=&nbsp;1</code>
+and <code>f_ref&nbsp;= c*s_ref</code>, which give the same results.
```

Comment:
Should this go to the info.revisions section? Otherwise it might be not clear what "same results" refers to?

Reply:
I thought that paragraph was self-contained; as it contains both formulations: `c*|s_rel|^n` and `f_ref*|s_rel/s_ref|^n` (and the defaults that make them the same). Readers may also prefer to see the simple formulation in the main documentation.

However, it may be appropriate to add it to the revisions as well.

Reply:
I fully agree to @HansOlsson